src/ADDomainServices/README.md

```diff
@@ -58,6 +59,9 @@ directive:
   - from: swagger-document
     where: $.definitions..trustPassword
     transform: $.format = "password"
+  - from: swagger-document
+    where: $.definitions..pfxCertificate
```

Comment:
should it be
$.definitions.LdapsSettings.pfxCertificate ?

Reply:
Yes. $.definitions..pfxCertificate means $.definitions.*.pfxCertificate. It works as well.